docs/using-the-jdbc-driver/using-plugins/UsingTheLimitlessConnectionPlugin.md

```diff
@@ -0,0 +1,42 @@
+# Using the Limitless Connection Plugin
+
+## What is Amazon Aurora Limitless Database?
+
+Amazon Aurora Limitless Database allows users to horizontally scale their Aurora Databases, handling millions of write transactions per second and managing petabytes of data. To learn more, please [read Amazon Aurora Limitless documentation](// TODO).
+
+## Why use the Limitless Connection Plugin?
+
+When connecting to an Amazon Aurora Limitless Database, clients will connect to the databases shard group endpoint, and be routed to a transaction router via Route 53.
+Unfortunately, Route 53 is limited in its ability to load balance, and can allow uneven work loads on transaction routers.
+The Limitless Connection Plugin addresses this by performing client-side load balancing with load awareness. 
+
+The Limitless Connection Plugin achieves this by periodically polling for load metric metadata of the transaction routers and caches it.
+When a new connection is made, the plugin will direct the connection to a transaction router selected using a weighted round-robin strategy.
+Routers with a higher load will be assigned a lower weight, and routers with a lower load will be assigned a higher weight.
+
+## How to use the Limitless Connection Plugin with the AWS JDBC Driver
+To enable the Endpoint Connection Plugin, add the plugin code `limitless` to the [`wrapperPlugins`](../UsingTheJdbcDriver.md#connection-plugin-manager-parameters) value, or to the current [driver profile](../UsingTheJdbcDriver.md#connection-plugin-manager-parameters).
+
+The URL used to connect to a Limitless database, should be the DB Shard Group URL.
```

Comment:
I think you need to be more imperative here

"Users have to use the shard URL to connect to a Limitless database."

Reply:
Done!